llvm/test/CodeGen/Xtensa/mul.ll

```diff
@@ -0,0 +1,672 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
I assume these are being generated with your own fork, because upstream LLVM does not support xtensa in update_llc_test_checks.py:

```
Traceback (most recent call last):
  File "llvm/utils/update_llc_test_checks.py", line 303, in <module>
    main()
  File "llvm/utils/update_llc_test_checks.py", line 193, in main
    scrubber, function_re = output_type.get_run_handler(triple)
  File "llvm/utils/UpdateTestChecks/asm.py", line 592, in get_run_handler
    raise KeyError("Triple %r is not supported" % (triple))
KeyError: "Triple 'xtensa' is not supported"
```

Please upstream this, as the whole point of using UTC is that others can rerun the scripts to update the tests automatically, but they can't do that if you've got non-upstreamed changes to UTC itself.